tests/unit/DateTime/DifferenceTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
```

Comment:
Test classess also should have namespaces. 
Please take a look at https://github.com/isolate-org/unit-of-work/tree/master/tests folder. Just remember to specify this namespace in composer.json file just like here: https://github.com/isolate-org/unit-of-work/blob/master/composer.json#L18